man/systemd-analyze.xml

```diff
@@ -1784,6 +1798,15 @@ boot  efi  exitrd  init  lib32	libx32	media	    opt  root  sbin  sys  usr  vmlin
         <xi:include href="version-info.xml" xpointer="v258"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--debugger=<replaceable>DEBUGGER</replaceable></option></term>
+
+        <listitem><para>When specified with the <command>unit-gdb</command> command, attach the given
+        service to the given debugger.</para>
```

Comment:
Please mention,
- when unspecified, `$SYSTEMD_DEBUGGER` will be used,
- when both unspecified, `gdb` will be used.

See man/coredump.xml.